locale/it/index.md

```diff
@@ -6,10 +6,10 @@ labels:
   download-for: Download per
   other-downloads: Altri Downloads
   other-lts-downloads: Altri Download LTS
-  other-stable-downloads: Altri Download Stabili
-  stable: Stabile
+  other-current-downloads: Altri Download Stabili
+  current: Corrente
```

Comment:
this is the exact kind of stuff I knew I would miss. Thanks for getting on it so quickly @fhemberger !!!